metadata_form/src/js/HierarchicalKeywordSelector.js

```diff
@@ -0,0 +1,237 @@
+import React from 'react'
+import axios from 'axios'
+import { TreeSelect, ConfigProvider, theme } from 'antd'

```

Comment:
I find TreeSelect from antd is really nice for the keyword selector, but I am a little bit concerned about how much bigger it makes the final js file (goes from about 800kb to 1.2 mb). This also makes the metadata form load a little slower. Not sure if there is some tree shaking I can do.